File: python/이어 붙인 수.py
def solution(num_list):
    answer = 0
    even = ''
    odd = ''
    for i in range(len(num_list)):
        if num_list[i] % 2 == 0:
            even += str(num_list[i])
        elif num_list[i] % 2 == 1 :
            odd += str(num_list[i])
    return int(even) + int(odd)

File: javascript/이어 붙인 수.js
const solution = (num_list) => {
    let answer = 0;
    let even = ''
    let odd = ''
    for(let i = 0; i < num_list.length; i++) {
        if(num_list[i]%2 == 0) {
            even += num_list[i].toString();
        }else {
            odd += num_list[i].toString();
        }
    }
    return parseInt(even) + parseInt(odd);
}